vignettes/articles/NMC Mar 2025.qmd

```diff
@@ -0,0 +1,363 @@
+
+## Agenda
+
+This workshop demonstrates potential uses (beyond their original collection purpose) for publicly available water quality data from Water Quality Portal (WQP). To start, participants will learn how to use EPA’s Tools for Automated Data Analysis (TADA) R Package to retrieve, wrangle, harmonize, quality check, visualize and analyze WQP data from multiple organizations.
+
+## Agenda
+
+Next, we will showcase how to bring in other web services and libraries for easy integration of additional hydrologic (e.g. USGS continuous sensor data), climatic and remote sensing data. We then plan to briefly touch on packages that can assist with building statistical models. Finally, we will demonstrate an example workflow for analyzing water quality by Assessment Unit's, which are state or tribal nation defined watershed or waterbody areas used for CWA assessments and reporting water quality conditions to EPA and the public.
+
+## Packages
+
+-   EPA: EPATADA, StreamCatTools
+
+-   USGS: dataRetrieval, nhdplusTools, hydroloom
+
+-   Fundamental geospatial packages such as sf, terra, leaflet and tmap
+
+# (15 mins) Preparing Water Quality Portal (WQP) data for analysis using EPATADA
+
+Let's dive into Green Bay, WI!
+
+## Install and load
+
+Install and load the remotes package.
+
+```{r, eval = F}
+if(!"remotes"%in%installed.packages()){
+install.packages("remotes")
+}
+
+library(remotes)
+```
+
+## Install and load TADA
+
+Install and load the [EPATADA R Package](https://github.com/USEPA/EPATADA) from GitHub.
+
+```{r, eval = F}
+remotes::install_github("USEPA/EPATADA", 
+                        ref = "develop", 
+                        dependencies = TRUE, 
+                        force = TRUE)
+
+library(EPATADA)
+```
+
+## Load required libraries
+```{r, include = F, eval = T}
+library(devtools)
+load_all()
+```
+
+## It's go time!
+
+Let's time our process.
+
+```{r}
+# Record start time
+start.time <- Sys.time()
+```
+
+## TADA Module 1: Water Quality Portal Data Discovery and Cleaning
+
+Additional functions and a more detailed example workflow is available here: https://usepa.github.io/EPATADA/articles/TADAModule1.html
+
+## Retrieve data from the WQP
+
+In this example, we will use EPA's How's My Waterway (HMW) application to find an applicable Hydrologic Unit Code (HUC) for our area of interest - the [Fox River, Green Bay, WI](https://mywaterway.epa.gov/community/040302040405/monitoring). Next, let's query the WQP using the identified HUC, state abbreviation, and a date range. In this example, we'll start by pulling all data available in the WQP for this HUC 12 for the last 5 years.
+
+WATERSHED: City of Green Bay-Fox River (040302040405)
+
+```{r}
+GreenBay_FoxRiver <- TADA_DataRetrieval(
+  statecode = "WI",
+  startDate = "2015-01-01",
+  endDate = "2024-12-30",
+  huc = c("040302040405"),
```

Comment:
I am not attached to using this particular HUC or the characteristics selected for the subset below. Feel free to edit if another works better